apps/projects/app/components/Panel/SubmitWork/SubmitWork.js

```diff
@@ -1,19 +1,147 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import styled from 'styled-components'
 
-const SubmitWork = ({ issues, rate, onSubmit }) => <div>Submit Work</div>
-
-SubmitWork.propTypes = {
-  /** array of issues to allocate bounties on */
-  issues: PropTypes.arrayOf(
-    PropTypes.shape({
-      id,
-      level,
-    })
-  ),
-  /** base rate in pennies */
-  rate: PropTypes.number,
-  onSubmit: PropTypes.func,
+import {
+  Field,
+  Text,
+  TextInput,
+  theme,
+  Info,
+  SafeLink,
+  IconAttention
+} from '@aragon/ui'
+
+import { Form, FormField, FieldTitle } from '../../Form'
+import { IconGitHub, CheckButton } from '../../Shared'
+
+class SubmitWork extends React.Component {
+
+  static propTypes = {
+    issue: PropTypes.object.isRequired
+  }
+
+  state = {
+    proof: '',
+    comments: '',
+    hours: 0,
+    ack1: false,
+    ack2: false
+  }
+
+  changeField = ({ target: { name, value } }) => this.setState({ [name]: value })
+  setAck1 = () => this.setState(prevState => ({ ack1: !prevState.ack1 }))
+  setAck2 = () => this.setState(prevState => ({ ack2: !prevState.ack2 }))
+
+  onSubmitWork = () => {
+    console.log('Submit', this.state)
+  }
+
+  canSubmit = () => !(this.state.ack1 && this.state.ack2 && this.state.proof)
```

Comment:
Can we validate that the hours worked field is populated and a number as well?

Reply:
Done.